org.eclipse.m2e.pde.ui/src/org/eclipse/m2e/pde/ui/editor/MavenTargetLocationWizard.java

```diff
@@ -165,17 +179,41 @@ private Button createCheckBox(Composite composite, String text) {
 				return box;
 			}
 
+			private void createIncludeCombo(Composite parent) {
+				Composite composite = new Composite(parent, SWT.NONE);
+				GridLayout layout = new GridLayout(2, false);
+				layout.horizontalSpacing = 20;
+				layout.marginWidth = 0;
+				layout.marginHeight = 0;
+				composite.setLayout(layout);
+				include = new ComboViewer(combo(new CCombo(composite, SWT.READ_ONLY | SWT.BORDER | SWT.FLAT)));
+				includeLabel = new Label(composite, SWT.NONE);
+				includeLabel.setText(Messages.MavenTargetLocationWizard_16);
```

Comment:
This could be a tool-tip instead.
At the moment the wizard appears a bit full.

Reply:
For sure it would be cool if someone would take the time and effort to raft a super-cool styled wizard for this, but at least it fulfills the needs, I'd rather try to invest the time in adding the functional bits here, so sorry for no eye-candy :-D